src/mjolnir/pbfgraphparser.cc

```diff
@@ -748,6 +748,14 @@ struct graph_callback : public OSMPBF::Callback {
       restriction.set_modes(kTruckAccess);
       osmdata_.access_restrictions.insert(
           AccessRestrictionsMultiMap::value_type(osmid_, restriction));
+    };
+     tag_handlers_["maxaxles"] = [this]() {
+      OSMAccessRestriction restriction;
+      restriction.set_type(AccessType::kMaxAxles);
+      restriction.set_value(std::stof(tag_.second) * 100);
```

Comment:
whats the 100 here for? i couldnt seem to find an entry in the wiki for this tag and taginfo shows its very sparingly populated BUT of the 79 ways that do have it, the values are 2 and 3.

Reply:
oh, also axles should be positive whole numbers i guess :D maybe `std::stoul` makes more sense

Reply:
Damn you are quick. Thanks for that! I was just creating the shell of what we need to gather and will submit it later, likely with a few questions attached. The calc there is not obviously unnecessary. 

Additionally, I agree that the taginfo:maxaxle(s) is sparingly used. However, I work for a trucking technology company, and we have seen issues in the past with a few places [(the Holland Tunnel specifically that we provided routing for)](https://valhalla.github.io/demos/routing/#loc=15,40.734104,-74.025385&locations=[{%22lat%22:40.7416970923627,%22lon%22:-74.04574133455752},{%22lat%22:40.73642941794871,%22lon%22:-74.00102354586123}]&costing=%22auto%22). 
I recently added the Holland Tunnel maxheight to OSM for security there. However, since there is no weight restriction for the HT, no axleLoad restriction, and only an axle count restriction, the routing is incorrectly allowing a >3 axle truck to drive through, when the Port Authority of NY/NJ doesn't allow this.  [HT OSM way](https://www.openstreetmap.org/edit#map=17/40.72823/-74.02449). [PANYNJ Restriciton PDF](https://www.panynj.gov/content/dam/bridges-tunnels/pdfs/truckers-guide.pdf)

I understand this is a one off circumstance, however, a simple fix to add the axle count to the restrictions should a decent safeguard for the trucking community. That being said, we will likely also be updating OSM data when we receive updates/comments from our truck driver community.

Reply:
oh yeah i think its great to add the axle restruction regardless of how well populated it is today, no worries about that! the extra `100` factor was why i was saying i couldnt find any information about it. i was imaging that maybe maxaxles were in the units `centiaxles` or something  ;) hence the scaling it back to integer but it seems to be the more obvious unit of axles itself haha